### DISCUSSION TITLE
Why does it run in the background constantly?

### DISCUSSION THREAD

**Discussion by zt64** [Mahmud0808/ColorBlendr#135] [Q&A] [answered]
Hi, I'm just wondering why color blendr runs in the background constantly. Shouldn't it just be applying the colors once and then nothing else?

**Comment by Mahmud0808** [comment, marked as answer]
It has to run in background to detect wallpaper change. If you don't need that, you can kill the app from background.

**Reply by zt64** [reply]
Ah ok, could this be added as an option though?

**Reply by Mahmud0808** [reply]
As I said, you can just restrict background activity.
